package.json

```diff
@@ -7,7 +7,8 @@
     "gulp": "gulp",
     "start": "gulp",
     "build": "NODE_ENV=production gulp build",
-    "test": "gulp test",
+    "test": "jest --coverage --collectCoverageFrom=src/**/*.js",
```

Comment:
I know you won't like it but I'd actually prefer to write a script file for that.
The reason is that you can then share the src/**/*.js from the config file that we have and we don't need to change it at several places if it ever does change.